Data Structures/Singly Circular Linked List/main.cpp

```diff
@@ -0,0 +1,96 @@
+/* ********* SINGLY LINK LIST ********* */
+#include<iostream>
+using namespace std;
+#include "list.cpp"
+int main()
+{
+	list myLinkList;
+	Node *ptr;
+	
+	// INSERTING VALUES IN AN EMPTY LIST
+	myLinkList.insert(10);
+	myLinkList.insert(20);
+	myLinkList.insert(30);
+	myLinkList.insert(40);
+
+	// PRINTING THE LIST
+	myLinkList.display();
+	cout << endl;
+	// PRINTING THE SIZE OF LIST

```

Comment:
if possible, please do start commenting with how this particular block of code would affect the overall execution of the code.